Field Service Management/README.md

```diff
@@ -0,0 +1,13 @@
+# Field Service Management
+This repository contains the Field Service Management blueprint and documentation. 
+
+To load from a VANTIQ Namespace locate the Projects --> Import button. Under Blueprints find Field Service Management and import the entire contents. 
```

Comment:
This content will be displayed after the user has already clicked Projects -> Import, and after they've selected Blueprints and selected "Field Service Management" as the blueprint to import, so I think this line can be skipped. The README should be more about what scenario the blueprint is for, and what components are included.

Reply:
I updated the Readme per your comments.